From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Basil Chew <basil.chew@intel.com>
Date: Fri, 24 Jun 2022 13:03:44 +0800
Subject: [PATCH] Use apt repository to install libssl

Change:
- switched installation of libssl1.1 to use focal apt repository
---
 scripts/setup_host.sh | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/scripts/setup_host.sh b/scripts/setup_host.sh
index 7edfc01..fc33f9f 100755
--- a/scripts/setup_host.sh
+++ b/scripts/setup_host.sh
@@ -316,9 +316,9 @@ function ubu_install_swtpm() {
 }
 
 function ubu_install_libssl() {
-    wget -N  http://archive.ubuntu.com/ubuntu/pool/main/o/openssl/libssl1.1_1.1.1l-1ubuntu1.3_amd64.deb
-    sudo dpkg -i libssl1.1_1.1.1l-1ubuntu1.3_amd64.deb
-    rm libssl1.1_1.1.1l-1ubuntu1.3_amd64.deb
+    sudo add-apt-repository -y 'deb http://security.ubuntu.com/ubuntu focal-security main'
+    sudo apt-get -y install libssl1.1
+    sudo add-apt-repository -y --remove 'deb http://security.ubuntu.com/ubuntu focal-security main'
 }
 
 function ubu_update_bt_fw() {
-- 
2.17.1

